src/encoded/commands/deploy.py

```diff
@@ -1077,7 +1077,14 @@ def hostname(value):
     args = parser.parse_args()
     # If needed, get ami.
     if not args.image_id:
-        current_ami_id = _fetch_ami_id()
+        if args.profile_name == "default":
+            print("Fetching development AMI.")
+            current_ami_id = _fetch_ami_id()
+        elif args.profile_name == "production":
+            print("Fetching production AMI.")
+            current_ami_id = _fetch_ami_id(key="ami-id/production/current_ami_id.txt")
+        else:
+            raise ValueError("--profile-name must be default or production")
```

Comment:
If it's not hard, maybe we could make this into `choices` in the argument parser?

Reply:
Yea, good point. That is the way.